.github/workflows/check_label.yml

```diff
@@ -68,6 +68,12 @@ jobs:
             echo "Expecting PR to not have the NeedsIssue label; please create a linked issue and remove the label."
             exit 1
           fi
+          if cat ${LABELS_JSON} | jq -r '.[].name ' | grep -q 'NeedsBackportReason' ; then
```

Comment:
We should quote all of these ${LABELS_JSON} variables. 

FWIW, we can also do this instead:
```
if [[ $(cat "${LABELS_JSON}" | jq -r '[.[] | select(.name | contains("good") or contains("revision"))] | length') -eq 2 ]]; then
    echo "Expecting PR to not have the NeedsBackportReason label; please add your justification to the PR description and remove the label."
    exit 1
fi
```

I'm not sure if it's more or less ugly though... 😆

Reply:
Definitely more 😄